api/v1alpha1/varnishcluster_webhook.go

```diff
@@ -115,6 +102,76 @@ func (in *VarnishCluster) ValidateDelete() error {
 	return nil
 }
 
+func validateCreateUpdate(in *VarnishCluster) error {
+	if in.Spec.Varnish != nil {
```

Comment:
Just curious why you named the parameter `in`? In most languages that is a reserved keyword, so may be confusing to some people.

Reply:
the original function definitions had that name for the argument so I kept using it. But I see how it can be confusing a bit, agree. I'll rename it to `vc` which is consistent with what we have in the rest of the code. Thanks.